external/deep-object-reid/torchreid_tasks/utils.py

```diff
@@ -400,6 +400,41 @@ def on_test_batch_end(self, batch=None, logs=None):
         self.update_progress_callback(self.get_progress())
 
 
+class OptimizationProgressCallback(TimeMonitorCallback):
+    """ Progress callback used for optimization using NNCF
+        There are four stages to the progress bar:
```

Comment:
```suggestion
        There are three stages to the progress bar:
```
Please change here and below.